pkg/sql/catalog/tabledesc/structured.go

```diff
@@ -2015,19 +2015,23 @@ func (desc *Mutable) addMutation(m descpb.DescriptorMutation) {
 // table descriptor returned should not include newly added constraints, which
 // is useful when passing the returned table descriptor to be used in
 // validating constraints to be added.
-const IgnoreConstraints = false
+const IgnoreConstraints = 1
+
+// IgnoreConstraintsAndPKSwaps is used in MakeFirstMutationPublic to indicate that the
+// table descriptor returned should include newly added constraints.
+const IgnoreConstraintsAndPKSwaps = 2
 
 // IncludeConstraints is used in MakeFirstMutationPublic to indicate that the
 // table descriptor returned should include newly added constraints.
-const IncludeConstraints = true
+const IncludeConstraints = 0
```

Comment:
Would you be willing to make a type and put these consts in a block?

Reply:
Also, can you move this definition to the `catalog` package so that the type can live there?

Reply:
ah, yeah, that's why I make it a type originally since it'd need to move to the interface package and I wasn't sure how all this would backport across all the catalog changes.

Reply:
Done